content/reference/admin/custom_sso/index.md

```diff
@@ -61,7 +61,12 @@ You will need to edit in the application integration you created to configure th
 If your SSO system supports both OIDC and SAML, we recommend using OIDC.
 {{< /alert >}}
 
-If your SSO system offers SAML v2.0 integration, it will typically publish its Identity Provider (IdP) entity descriptor on a specific URL. To configure the SSO integration, we will need this IdP entity descriptor URL.
+If your SSO system offers SAML v2.0 integration, it will typically publish its Identity Provider (IdP) entity descriptor on a specific URL.
+
+To configure the SSO integration, we will need:
+- Identity provider SAML entity descriptor
+- Attributes names for: first name, last name and email.
+- Principal type and format _(commonly subject NameID)_
```

Comment:
There are two different aspects here:
- Using a Principal Type other than "Subject NameID": it's technically possible, but I wonder if it actually happens? But I guess it does deserve some explanation.
- the NameID policy format to specify in authentication requests sent to the IdP: in most cases we can just leave it unspecified (`urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified`), and leave this to the IdP. But maybe we'll need to be more specific when integration with some IdPs, so may deserve an explanation too.

In both cases, I think we need to give a real explanation and not just mention the names. And make it clear that there is a default case that should work for most people (based on my understanding of SAML, I may be wrong... 😅).

Here's my suggestion for a paragraph about this:

> By default, our system will rely on the subject NameID to uniquely and consistently identify each user. We will also, by default, leave the NameID policy format unspecified (`urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified`), letting the IdP free to choose how to populate the subject NameID. If needed we can rely on another attribute instead of the NameID (it needs to be unique and immutable), or specify a particular NameID policy format.